autogpt/memory/milvus.py

```diff
@@ -1,4 +1,7 @@
 """ Milvus memory storage provider."""
+import re
+
+from config import Config
```

Comment:
`ModuleNotFoundError` is produced here in my python env.
`from autogpt.config import Config` is right.

Reply:
And this is why we need unit tests :')

Can you submit a PR to fix? I'll merge it as soon as I see it.

Reply:
ok, I submit a new PR to fix it now.
It seems that `pymilvus` has been removed from requirement.
Unittest was skipped, I will submit a new PR to fix this problem in future.

Reply:
@Pwuts #3036 here :)

Reply:
Please tag me when you submit a PR to fix the test :)

Reply:
OK